### ISSUE TITLE
Documentation needed.

### ISSUE THREAD

**Issue by yohnniebabe** [Pix4D/three-potree-loader#118]
Thanks for the wonderful library!

Do you have some documentation on how to set the camera controls and / or some reference to set the other settings?

Any help would be appreciated!

**Comment by Washipp** [comment]
Well its built on threejs so you can use it to set the camera controls. https://threejs.org/docs/ 

Regarding a documentation as far as I know you only have the code and this example: https://github.com/pnext/three-loader/tree/master/example

Of course you can look through Github to see other examples.

**Comment by yohnnylim** [comment]
Yeah thanks for the reply, I couldn't find any other examples on using this package, the only example is the one in the example folder. Looks like alot of the potree functions were not migrated into this package, liek Viewer.fitToScreen() this method can't be used in this package.
